datavalid.yml

```diff
@@ -462,5 +467,33 @@ schemas:
         unique: true
         no_na: true
       - name: agency
+  property_claims:
+    columns:
+      - name: property_claims_uid
+        description: >
+          unique identifier of award.
+        unique: true
+        no_na: true
+      - name: uid
+      - name: agency
+      - name: claim_id
+      - name: jurisdiction
+      - name: accident_desc
+      - name: accident_location
+      - name: total_paid
+      - name: claim_day
+      - name: claim_month
+      - name: claim_year
+      - name: receive_day
+      - name: receive_month
+      - name: close_day
+      - name: close_month
+      - name: close_year
```

Comment:
Do we need all these date columns when most dates should end up in the events table?

Reply:
Thanks.